tests/ui/conftest.py

```diff
@@ -58,3 +60,33 @@ def user(base_url, fxa_account, jwt_token):
     assert requests.codes.created == r.status_code
     user.update(r.json())
     return user
+
+
+@pytest.yield_fixture(scope='session')
+def firefox_path(tmpdir_factory):
```

Comment:
If a test requires a specific minimum version of Firefox then I feel like we should indicate it in those tests using a marker. I don't think it should be a `firefox_path` override though, but should determine the version of Firefox based on the path (or session capabilities if it's a remote instance) and compare it with what we're expecting. Let's leave this for a patch that requires a specific Firefox version though (unless there are tests in here that already require 48+?)

I'm thinking something like:

```python
@pytest.mark.firefox(version='>=48')
def test_install_extension_from_discovery_pane(base_url, selenium, discovery_pane):
   ...
```